src/activities/menu/Menu.qml

```diff
@@ -77,6 +78,8 @@ ActivityBase {
 
     // @cond INTERNAL_DOCS
     property string url: "qrc:/gcompris/src/activities/menu/resource/"
+    property int  cursorPosition: 0
+    property  string inputText: ""
```

Comment:
You can set it as an alias of the textInput.text (or the opposite), this way, it will be automatically updated.